libkpod/remove.go

```diff
@@ -16,8 +16,13 @@ func (c *ContainerServer) Remove(container string, force bool) (string, error) {
 	}
 	ctrID := ctr.ID()
 
-	cState := c.runtime.ContainerStatus(ctr)
-	if cState.Status == oci.ContainerStateCreated || cState.Status == oci.ContainerStateRunning {
+	cStatus := c.runtime.ContainerStatus(ctr)
+	switch cStatus.Status {
+	case oci.ContainerStatePaused:
+		if !force {
+			return "", errors.Errorf("cannot remove paused container %s", ctrID)
+		}
```

Comment:
Thought: should we try unpausing before removing if force is set?

Reply:
Unpausing would mean you need to do a containerstop, I believe.  We should try this out to see if a paused container can be removed with a --force flag

Reply:
I've created a test in the bats file, seems to work fine as is.  i.e. the container is removed and it's no longer there after.